pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -118,7 +118,10 @@ type rateLimitedRoundTripper struct {
 }
 
 func (rl *rateLimitedRoundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
+	startTime := time.Now()
 	rl.limiter.Accept()
+	// TODO: Reduce verbosity once #26119 is fixed.
+	glog.V(1).Infof("GCE api call: %s %s (throttled for %v)", req.Method, req.URL.String(), time.Now().Sub(startTime))
```

Comment:
It seems that e.g. in GKE by default we don't log on --v=1. So I changed it temporarily to V(0).
I will reduce the verbosity once we understand this.